src/routes/stats/StatsView.vue

```diff
@@ -84,7 +84,7 @@
           <h2 class="text-h2 mt-8 mb-4">
             {{ t('stats.siteUsage') }}
           </h2>
-          <StatsUsageChart :season="selectedSeason" />
+          <StatsUsageChart v-if="selectedSeason.gameCounts>0" :season="selectedSeason" />
```

Comment:
Looking more closely at the shape of the season data, this conditional is incorrect, because the `gameCounts` is an array of integers, not a single integer. Because this makes it more complicated, let's pull this into its own computed property called `showStatsUsageChart`, which can essentially return false if either the array is empty, or every element in the array is zero. It could look like this:

```
computed: {
  showStatsUsageChart() {
    const gameCounts = this.selectedSeason?.gameCounts ?? [];
    // Show unless gameCounts is empty or all zeros
    return gameCounts.length > 0 && !gameCounts.every(gameCount => gameCount === 0)
  }
}
```

Reply:
I have tried this and the tests still fail